pyquil/api/tests/test_compiler.py

```diff
@@ -100,6 +100,28 @@ def test_http_compilation(compiler):
     assert compilation_result.program == SIMPLE_RESPONSE["program"]
 
 
+def test_invalid_protocol():
```

Comment:
This looks like it might be from the previous PR where UserMessageError behavior was changed